### DISCUSSION TITLE
How do I disable powershell pop-up during updating

### DISCUSSION THREAD

**Discussion by Ag0v0** [tauri-apps/tauri#6037] [Q&A]
I don't want powershell to pop up when I apply the update, so how do I disable it？😀

![image](https://user-images.githubusercontent.com/63031576/212066253-71354b5c-982d-4cd4-b6bd-45ffa6bcee88.png)

**Comment by JonasKruckenberg** [comment]
Is your app build in debug mode? The powershelgl prompt standard for debug builds, but shouldn't really show up for production ones
